### PR TITLE
new google analytics dimensions

### PR DESCRIPTION
**Pull request by AWare**
@paulbrown1982 
@ajosephides

### CODE REVIEW

File: frontend/assets/javascripts/src/modules/analytics/ga.es6 [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            }, i[r].l = 1 * new Date();
+        a = s.createElement(o),
+            m = s.getElementsByTagName(o)[0];
+        a.async = 1;
+        a.src = g;
+        m.parentNode.insertBefore(a, m)
+    })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
+    /*eslint-enable */
+
+    ga('create', guardian.googleAnalytics.trackingId, {
```
Comment (ajosephides) [inline frontend/assets/javascripts/src/modules/analytics/ga.es6]: Can we give the tracker, even though there is only one, a name.
Just to be explicit in what we call.

Comment (paulbrown1982) [inline frontend/assets/javascripts/src/modules/analytics/ga.es6]: You can do it like I did in Subs @AWare https://github.com/guardian/subscriptions-frontend/blob/master/assets/javascripts/modules/analytics/ga.js you should use the same name ideally too.

**Comment by AWare** [comment]
@ajosephides fixed, raising c-f pr for the tracker name

**Comment by paulbrown1982** [comment]
You need to prefix all the actions with the name too @AWare 
e.g.
        ga('membershipPropertyTracker.require', 'linkid');
        ga('membershipPropertyTracker.set', 'dimension5', 'subscriptions');

File: frontend/assets/javascripts/src/modules/analytics/ga.es6 [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    ophanPageViewId: 'dimension3',
+    ophanBrowserId: 'dimension4',
+    platform: 'dimension5',
+    identityId: 'dimension6',
+    isLoggedOn: 'dimension7'
+};
+
+function create(){
+    const tracker = 'membershipPropertyTracker';
+    ga('create', guardian.googleAnalytics.trackingId, {
```
Comment (paulbrown1982) [inline frontend/assets/javascripts/src/modules/analytics/ga.es6]: will ga be available here?

Comment (AWare) [inline frontend/assets/javascripts/src/modules/analytics/ga.es6]: It will be available, but it's potentially an unpleasant amount of global nonsense.

**Comment by AWare** [comment]
https://github.com/guardian/contributions-frontend/pull/65

**Comment by paulbrown1982** [comment]
👍

**Pull request merged**